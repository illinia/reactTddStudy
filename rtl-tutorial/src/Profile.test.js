import React from 'react';
import { render } from '@testing-library/react';
import Profile from './Profile';

describe('<Profile />', () => {
  it('matches snapshot', () => {
    const utils = render(<Profile username="test" name="test1" />)
    expect(utils.container).toMatchSnapshot();
  });
  it('shows the props correctly', () => {
    const utils = render(<Profile username="test" name="test1" />);
    utils.getByText('test');
    // utils.getByText('test1');
    // utils.getByText(/te/);
  });
});