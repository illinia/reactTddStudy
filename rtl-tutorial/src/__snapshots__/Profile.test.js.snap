// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> matches snapshot 1`] = `
<div>
  <div>
    <b>
      test
    </b>
    <span>
      (
      test1
      )
    </span>
  </div>
</div>
`;
